docs/data.md

```diff
@@ -0,0 +1,131 @@
+# Data dictionary
+
+<sup>_Quick links:
+&nbsp; [A](#a)
+&nbsp; [B](#b)
+&nbsp; [C](#c)
+&nbsp; [D](#d)
+&nbsp; [E](#e)
+&nbsp; [F](#f)
+&nbsp; [I](#i)
+&nbsp; [M](#m)
+&nbsp; [N](#n)
+&nbsp; [O](#o)
+&nbsp; [P](#p)
+&nbsp; [Q](#q)
+&nbsp; [R](#r)
+&nbsp; [S](#s)
+&nbsp; [T](#t)
+&nbsp; [U](#u)
+&nbsp; [V](#v)
+_</sup>
+
+The following table describes the columns available within the exported data:
+
+| Name                              | Type    | Source ([learn more](#source)) | Version | Description                                                                                                                                                                                                                                                            |
+| --------------------------------- | ------- | ------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <a name="a"></a>AccountName       | String  | Storage: ms-cm-exports         | 0.0.1+  | Name of the identity responsible for billing for this subscription. This is your EA enrollment account owner or MOSA account admin. Not applicable to MCA.                                                                                                             |
```

Comment:
Version `0.0.1+` indicates that this column is available in the first toolkit release. I mostly added this for the future as we introduce new and/or deprecate columns.